display_list/geometry/dl_region_unittests.cc

```diff
@@ -13,7 +13,7 @@ namespace flutter {
 namespace testing {
 
 TEST(DisplayListRegion, EmptyRegion) {
-  DlRegion region({});
+  DlRegion region(std::vector<SkIRect>{});
```

Comment:
Just a thought: Should it have a default constructor?

Reply:
It should and has. I'll fix the test.